internal/helm/helm.go

```diff
@@ -124,13 +124,15 @@ func setupTillerConnection(settings *environment.EnvSettings, config *rest.Confi
 }
 
 func (c *Client) Close() {
-	if c.forwarder != nil {
+	if c != nil && c.forwarder != nil {
```

Comment:
Why check the pointer to the object?

Reply:
It is not really necessary, I agree that it is not good to check this in all pointer receiver methods.

But in this specific method I thought it is a good idea, because often this method will be called during cleanup or disposal of resources and it is likely to happen that `Close()` is called after the receiver is set to `nil`.